### ISSUE TITLE
[Macro] Adding back Macros to Mathjax config

### ISSUE THREAD

**Issue by Miniland1333** [mathjax/mathjax-v3#201]
I did some testing with the Mathjax v3 Beta, and it served as an almost drop-in replacement for our existing version! However, I noticed that the Macros I tried to define in MathJaxConfig are no longer applying.  It may just be a configuration error, but if Macros in the config have not been implemented, let me know both where the Macro-config code should be implemented and where the code was defined in v2 as a reference!

```javascript
let front = 'Testing ';
MathJaxConfig = {
  TeX: {
     Macros: {
        test: ["{"+front+" #1}",1]
     }
  }
};
```

The macro I want is quite simple, so hopefully it isn't too much of a hassle to set up.

**Comment by zorkow** [comment]
Thanks for the feedback. It appears that the Macros option is currently not properly processed. I am working on it.

**Comment by Miniland1333** [comment]
Hey @zorkow! Is there anything I can contribute to help with this issue? I don't know where in this repo the code for Macros option processing is.

**Comment by dpvc** [comment]
I've already made a pull request that makes the macros option available again (but `macros` rather than `Macros` in keeping with how v3 handles capitalization).  See #222.  I'm hoping to have a beta.4 out this coming week that will include all the new features, including macros configuration.

**Comment by zorkow** [comment]
Tnanks for the poke @Miniland1333 .
I've finished reviewing the PR today and it should land on master shortly.

**Comment by dpvc** [comment]
I have checked on the only issue that was rated in the PR, and it looks like everything is ready.  As soon as Volker approves the PR, I'll merge it.

**Comment by Miniland1333** [comment]
Closed with #222. Thanks for your help!
